src/react-components/media-browser.js

```diff
@@ -138,6 +138,7 @@ class MediaBrowser extends Component {
     if (!this.props.onMediaSearchResultEntrySelected) return;
     this.props.onMediaSearchResultEntrySelected(entry);
     this.close();
+    return false;
```

Comment:
Would be nice to use `e.preventDefault()` instead to be more explicit.

Reply:
ah yes, i'm too old school